### PR TITLE
Documentation for the x-goto functions

### PR DESCRIPTION
**Pull request by brandelune**
Documentation for the functions described here:
https://github.com/omegat-org/omegat/pull/370

I've also changed the names of the menu items following my comment in the above PR.

### CODE REVIEW

**Review by kosivantsov — CHANGES_REQUESTED** [review]
`doc_src/en/HowTo_UseTM.xml`
> <itemizedlist>
	  <listitem>
		<para>either ppen the old project and put the new source

Capitalization and typo

> <listitem>
		<para>or put the reference memories that

Capitalization

**Comment by kosivantsov** [comment]
Other than the requested changes, I'm very happy with this PR

**Review by miurahr — CHANGES_REQUESTED** [review]
Could you fix a typo?

**Comment by brandelune** [comment]
I should be good now.

**Review by miurahr — APPROVED** [review]

File: doc_src/en/OmegaT5_ProjectFolder.xml [hunk 1]
```diff
@@ -261,7 +262,13 @@
 			  project translation memory.</para>
 			</listitem>
 		  </orderedlist>
-		  
+
+		  <para>Use the navigation menus found in <link linkend="menus.goto"
+		  endterm="menus.goto.title"/><link
+		  linkend="menus.goto.auto.populated.segments"
+		  endterm="menus.goto.auto.populated.segments.title"/> to navigate the
+		  segments that have been inserted.</para>
```
Comment (Kazephil) [inline doc_src/en/OmegaT5_ProjectFolder.xml:270]: "...to navigate to the inserted segments."

File: doc_src/en/OmegaT5_ProjectFolder.xml [hunk 2]
```diff
@@ -278,15 +285,25 @@
 
 		  <para>Translations from TMX files placed in this folder are
 		  automatically inserted in matching segments without the automatic
-		  prefix, making it unnecessary to confirm these segments. Translations
-		  coming from this folder are considered to be <emphasis>more</emphasis>
-		  reliable than the current project memory.</para>
+		  prefix, making it unnecessary to confirm these segments.</para>
```
Comment (Kazephil) [inline doc_src/en/OmegaT5_ProjectFolder.xml:288]: I think either "without the fuzzy match prefix" or "without any prefix" would be better than "without the automatic prefix".

**Review by Kazephil — CHANGES_REQUESTED** [review]

**Comment by Kazephil** [comment]
A bit nitpicky in places, but I've proposed changes that I think make the text flow better.
Everything else looks good!

**Comment by brandelune** [comment]
I'm on it.

**Comment by brandelune** [comment]
I should be good now. @Kazephil can you take a last look ?

**Review by kosivantsov — APPROVED** [review]

File: doc_src/en/HowTo_UseTM.xml [hunk 3]
```diff
@@ -74,21 +74,30 @@
...
+		the <link linkend="project.folder.tm" endterm="project.folder.tm.title"/>
+		folder of the new one.</para>
+
+		<note>
+		  <para>That’s the preferred method if you want to start your
+		  translation from scratch.</para>
+		</note>
+
+		<para>Depending on how reliable the memory is, you can prefer to put it
+		in either of the following subfolder:</para>
```
Comment (Kazephil) [inline doc_src/en/HowTo_UseTM.xml:100]: Last look:

Change "in either of the following subfolder:" to "in one of the following subfolders:"

File: doc_src/en/OmegaT5_ProjectFolder.xml [hunk 4]
```diff
@@ -325,7 +342,13 @@
 			  </itemizedlist>
 			</listitem>
 		  </orderedlist>
-		  
+
+		  <para>Use the navigation menus found in <link linkend="menus.goto"
+		  endterm="menus.goto.title"/><link
+		  linkend="menus.goto.auto.populated.segments"
+		  endterm="menus.goto.auto.populated.segments.title"/> to navigate the
+		  segments that have been inserted.</para>
```
Comment (Kazephil) [inline doc_src/en/OmegaT5_ProjectFolder.xml:350]: Small typo: "to navigate the" --> "to navigate to the"

**Review by Kazephil — CHANGES_REQUESTED** [review]
@brandelune 
Two more small changes, and you're good to go.

**Review by Kazephil — APPROVED** [review]
Good to go!

**Comment by miurahr** [comment]
The changes are conflict for rebase, so I just merged into master. Please don't merge master in mid of commit history, otherwise, you should use ` git rebase master`on topic branch.

**Comment by brandelune** [comment]
> The changes are conflict for rebase, so I just merged into master. Please don't merge master in mid of commit history, otherwise, you should use ` git rebase master`on topic branch.

I'm sorry. There are still things I do not understand regarding git. I don't remember why I merged at that point.

**Pull request merged**